plugins/nginx/src/ngx_http_waflz_module.c

```diff
@@ -0,0 +1,572 @@
+//: ----------------------------------------------------------------------------
+//: Copyright (C) 2018 Verizon.  All Rights Reserved.
+//: All Rights Reserved
+//:
+//: \file:    ngx_http_waflz_module.c
+//: \details: TODO
+//: \author:  Devender Singh
+//: \date:    09/28/2018
+//:
+//:   Licensed under the Apache License, Version 2.0 (the "License");
+//:   you may not use this file except in compliance with the License.
+//:   You may obtain a copy of the License at
+//:
+//:       http://www.apache.org/licenses/LICENSE-2.0
+//:
+//:   Unless required by applicable law or agreed to in writing, software
+//:   distributed under the License is distributed on an "AS IS" BASIS,
+//:   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//:   See the License for the specific language governing permissions and
+//:   limitations under the License.
+//:
+//: ----------------------------------------------------------------------------
+//: ----------------------------------------------------------------------------
+//: includes
+//: ----------------------------------------------------------------------------
+#include "stdio.h"
+#include <ngx_config.h>
+#include <ngx_core.h>
+#include <ngx_http.h>
+#include "ngx_http_waflz_module.h"
+
+static ngx_int_t ngx_http_waflz_init(ngx_conf_t *cf);
+//: ----------------------------------------------------------------------------
+//: routines to allocate and init main conf
+//: ----------------------------------------------------------------------------
+static void * ngx_http_waflz_create_main_conf(ngx_conf_t *cf);
```

Comment:
i think pointer * can be either near void/type or near function.. i always do near type but reed does it near function..google c++ std is near type i think. so whichever you prefer.. we can start making it consistent in new files

Reply:
thx, changed it to nginx style guide. which is near func/var